app/models/bookingsync_portal/rental.rb

```diff
@@ -2,8 +2,11 @@ class BookingsyncPortal::Rental < ActiveRecord::Base
   self.table_name = 'rentals'
 
   belongs_to :account, class_name: BookingsyncPortal.account_model
-  has_one :connection, class_name: BookingsyncPortal.connection_model, dependent: :destroy
-  has_one :remote_rental, class_name: BookingsyncPortal.remote_rental_model, through: :connection
+  has_one    :connection, class_name: BookingsyncPortal.connection_model, dependent: :destroy
+  has_one    :remote_rental, class_name: BookingsyncPortal.remote_rental_model, through: :connection
+  has_one    :availability, class_name: BookingsyncPortal.availability_model, dependent: :destroy
```

Comment:
So no change required? I am not sure about schema, would migrations be required even for unused models?